docker/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Copyright 2016-2020 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+set -eux
+
+# This script decides based on $RELEASE_VERSION whether to build or download the binaries we need.
+if [ "$RELEASE_VERSION" = "master" ]
+then
+  echo "*** Building from source..."
+  # Use the same command as we suggest in zipkin-server/README.md
+  #  * Uses mvn not ./mvnw to reduce layer size: we control the Maven version in Docker
+  (cd /code; mvn -q --batch-mode -DskipTests -Dlicense.skip=true --also-make -pl zipkin-server package)
```

Comment:
consider ` mvn -T1C -q --batch-mode -DskipTests -Dlicense.skip=true --also-make -pl zipkin-server package` for parallel builds.

Reply:
sure. trying now. thanks for the tip

Reply:
significantly faster thanks!